twitch/api/users.py

```diff
@@ -12,6 +12,10 @@ class Users(TwitchAPI):
     def get(self):
         response = self._request_get('user')
         return User.construct_from(response)
+      
+    def get_by_name(self, user_id):
+        response = self._request_get('users?login={}'.format(user_id))
```

Comment:
`_request_get` accepts a url an params, where params attribute is a dictionary containing get parameters.
So this like should be something like
```
response = self._request_get('users', params={'login': user_name})
```

Reply:
I just copied the syntax from get_by_id() more or less. When your code creates a URL like this
`https://api.twitch.tv/kraken/users?login=<user IDs>`
this should be fine. However, how would you create a URL with multiple user names like this
`https://api.twitch.tv/kraken/users?login=dallas,dallasnchains`

Reply:
Ah its that kind of URL param.
I'd suggest then doing something like
```
def get_by_name(self, user_ids):
    if not isinstance(user_ids, list):
        user_ids = list(user_ids)
    params = {
        'login': ','.join(user_ids)
    }
    response = self._request_get('users', params=params)
```

Reply:
In python 3.7.3 `isinstance("user_name", list)` is true.

To make your suggestion work:
```python
def get_by_name(self, user_ids):
        if isinstance(user_ids, str):
            user_ids = [user_ids]
        params = {
            'login': ','.join(user_ids)
        }
        response = self._request_get('users', params=params)
        return User.construct_from(response)
```